funnel/views/errors.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import unicode_literals
+
+from flask import current_app, redirect, request
+from werkzeug import exceptions
+from werkzeug.routing import MethodNotAllowed, NotFound, RequestRedirect
+
+from coaster.views import render_with
+
+from .. import app
+from . import baseframe_translations
+
+
+@app.errorhandler(exceptions.TooManyRequests)
+@render_with('429.html.jinja2', json=True)
+def error429(e):
+    baseframe_translations.as_default()
+    return {'error': "429 Too Many Requests"}, 429
+
+
+@app.errorhandler(exceptions.Gone)
+@render_with('410.html.jinja2', json=True)
+def error410(e):
+    baseframe_translations.as_default()
+    return {'error': "410 Page now gone"}, 410
+
+
+@app.errorhandler(exceptions.MethodNotAllowed)
+@render_with('405.html.jinja2', json=True)
+def error405(e):
+    baseframe_translations.as_default()
+    return {'error': "405 Method Not Allowed"}, 405
```

Comment:
These methods should be sorted by numeric code, so 405 after 404, and 410 after that.